CHANGELOG.md

```diff
@@ -6,6 +6,33 @@ This changelog adheres to [Keep a CHANGELOG](http://keepachangelog.com/).
 
 ## [Unreleased]
 
+## [0.9.0] - 2017-01-06
+### Added
+- Support for reading vmpooler tokens from a 
+  [vmfloaty](https://github.com/briancain/vmfloaty) config file.
+  The new order of precedence for defining vmpooler tokens is:
+    - `VMPOOLER_TOKEN` environment variable
+    - Reading it from the file `~/.vmpooler-token`
+    - Reading it from the file `~/.vmfloaty.yml`
+- A Rakefile to help ease common development tasks, like running spec tests, code coverage,
+  and Rubocop. `bundle exec rake -T` away!
+- Added `--only-build` flag, allowing developers to build a subset of components 
```

Comment:
That's probably going to be closed. But [c6cf6f](https://github.com/puppetlabs/vanagon/commit/c6cf6f9f5d2c6c4ff75ac227b64d7711cd46e244) is already merged.